### PR TITLE
Pull steps to start tendermint node into a shell script for easier lo…

### PR DESCRIPTION
**Pull request by janedegtiareva**
…cal dev.

### CODE REVIEW

File: scripts/waitonserver.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 echo 'waiting on healthcheck' >&2
 
-for _ in {1..6}; do
+for _ in {1..100}; do
```
Comment (bowenwang1996) [inline scripts/waitonserver.sh]: Maybe sleep for some time?

File: scripts/start_tendermint.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
```
Comment (nearmax) [inline scripts/start_tendermint.sh]: The name of the script is misleading. It starts more than just tendermint, it starts nearmint app, too. Rename to run_nearmint.sh (to also be consistent with `ops`)?

File: scripts/waitonserver.sh (hunk 1 above)
Comment (vgrichina) [reply scripts/waitonserver.sh]: there is `sleep 5` below already, sleeping for longer means there's gonna be delay between start and when it's detected

Comment (bowenwang1996) [reply scripts/waitonserver.sh]: Oh did't see that

File: scripts/start_tendermint.sh (hunk 2 above)
Comment (janedegtiareva) [reply scripts/start_tendermint.sh]: done

**Pull request merged**